src/scaffoldmaker/meshtypes/meshtype_3d_lung1.py

```diff
@@ -98,20 +99,29 @@ def generateBaseMesh(cls, region, options):
 
         lungGroup = AnnotationGroup(region, get_lung_term("lung"))
         leftLungGroup = AnnotationGroup(region, get_lung_term("left lung"))
-        #rightLungGroup = AnnotationGroup(region, get_lung_term("right lung"))
-        annotationGroups = [leftLungGroup, lungGroup]
+        rightLungGroup = AnnotationGroup(region, get_lung_term("right lung"))
+        annotationGroups = [leftLungGroup, lungGroup, rightLungGroup]
 
         lungMeshGroup = lungGroup.getMeshGroup(mesh)
         leftLungMeshGroup = leftLungGroup.getMeshGroup(mesh)
-        #rightLungMeshGroup = rightLungGroup.getMeshGroup(mesh)
+        rightLungMeshGroup = rightLungGroup.getMeshGroup(mesh)
 
         if isHuman:
             lowerLeftLungGroup = AnnotationGroup(region, get_lung_term("lower lobe of left lung"))
             lowerLeftLungMeshGroup = lowerLeftLungGroup.getMeshGroup(mesh)
             upperLeftLungGroup = AnnotationGroup(region, get_lung_term("upper lobe of left lung"))
             upperLeftLungMeshGroup = upperLeftLungGroup.getMeshGroup(mesh)
+            lowerRightLungGroup = AnnotationGroup(region, get_lung_term("lower lobe of right lung"))
+            lowerRightLungMeshGroup = lowerRightLungGroup.getMeshGroup(mesh)
+            upperRightLungGroup = AnnotationGroup(region, get_lung_term("upper lobe of right lung"))
+            upperRightLungMeshGroup = upperRightLungGroup.getMeshGroup(mesh)
+            middleRightLungGroup = AnnotationGroup(region, get_lung_term("middle lobe of right lung"))
+            middleRightLungMeshGroup = middleRightLungGroup.getMeshGroup(mesh)
             annotationGroups.append(lowerLeftLungGroup)
             annotationGroups.append(upperLeftLungGroup)
+            annotationGroups.append(lowerRightLungGroup)
+            annotationGroups.append(upperRightLungGroup)
+            annotationGroups.append(middleRightLungGroup)
```

Comment:
Put in triple:
get group
get meshGroup
add to annotationGroups
So others can copy more easily.